artifactory/commands/transferconfig/configxmlutils/federatedrepos.go

```diff
@@ -21,87 +15,47 @@ func ReplaceUrlsInFederatedrepos(configXml, sourceBaseUrl, targetBaseUrl string)
 	}
 	results := ""
 	prefix, content, suffix := splitXmlTag(configXml, xmlTagIndices, 0)
-	federatedRepositories, err := fixFederatedRepositories(content, sourceBaseUrl, targetBaseUrl)
+	federatedRepositories, err := fixFederatedRepositories(content)
 	if err != nil {
 		return "", err
 	}
 	results += prefix + federatedRepositories + suffix
 	return results, nil
 }
 
-func fixFederatedRepositories(content, sourceBaseUrl, targetBaseUrl string) (string, error) {
+func fixFederatedRepositories(content string) (string, error) {
 	xmlTagIndices, exist, err := findAllXmlTagIndices(content, `federatedRepository`, false)
 	if err != nil {
 		return "", err
 	}
 	if !exist {
 		return content, nil
 	}
+	federatedMembersRemoved := false
 	results := ""
 	for i := range xmlTagIndices {
 		prefix, content, suffix := splitXmlTag(content, xmlTagIndices, i)
-		federatedRepository, err := fixFederatedRepository(content, sourceBaseUrl, targetBaseUrl)
+		federatedRepository, removed, err := removeFederatedMembers(content)
 		if err != nil {
 			return "", err
 		}
 		results += prefix + federatedRepository + suffix
+		federatedMembersRemoved = federatedMembersRemoved || removed
 	}
-	return results, nil
-}
-
-func fixFederatedRepository(content, sourceBaseUrl, targetBaseUrl string) (string, error) {
-	xmlTagIndices, exist, err := findAllXmlTagIndices(content, `federatedMembers`, false)
-	if err != nil {
-		return "", err
-	}
-	if !exist {
-		return content, nil
-	}
-	results := ""
-	for i := range xmlTagIndices {
-		prefix, content, suffix := splitXmlTag(content, xmlTagIndices, i)
-		federatedMembers, err := fixFederatedMembers(content, sourceBaseUrl, targetBaseUrl)
-		if err != nil {
-			return "", err
-		}
-		results += prefix + federatedMembers + suffix
-	}
-	return results, nil
-}
-
-func fixFederatedMembers(content, sourceBaseUrl, targetBaseUrl string) (string, error) {
-	xmlTagIndices, exist, err := findAllXmlTagIndices(content, `federatedMember`, false)
-	if err != nil {
-		return "", err
-	}
-	if !exist {
-		return content, nil
-	}
-	results := ""
-	for i := range xmlTagIndices {
-		prefix, content, suffix := splitXmlTag(content, xmlTagIndices, i)
-		federatedMember, err := fixFederatedMemberUrl(content, sourceBaseUrl, targetBaseUrl)
-		if err != nil {
-			return "", err
-		}
-		results += prefix + federatedMember + suffix
+	if federatedMembersRemoved {
+		log.Info("☝️  All federated members have been excluded from your federated repositories. Please configure your federation in the target server repositories.")
```

Comment:
All federated members have been excluded from your federated repositories. Please configure your federation in the target server repositories.
-->
Your Federated Repositories have been ttransferred to your target instabce, but their members have been removed on the target. You should add members to your Federated repositories on your target instance as described here - https://www.jfrog.com/confluence/display/JFROG/Federated+Repositories